source/pages/Settings/PrivateKey.tsx

```diff
@@ -91,14 +95,35 @@ const PrivateKeyView = () => {
         <CopyCard
           className="my-3"
           onClick={
-            valid ? () => copyText(String(activeAccount?.xprv)) : undefined
+            valid
+              ? () =>
+                  copyText(
+                    String(
+                      isSyscoinChain
+                        ? CryptoJS.AES.decrypt(
+                            activeAccount?.xprv,
+                            pwd
```

Comment:
```suggestion
                            form.getFieldValue('password')
```
 form.getFieldValue can get the input value so we don't need to create a new state for pwd